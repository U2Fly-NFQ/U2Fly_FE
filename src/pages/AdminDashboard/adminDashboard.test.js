describe('Admin dashboard Layout', () => {
  it('renders correctly', () => {
    // const tree = renderer
    //   .create(
    //     <I18nextProvider i18n={i18n}>
    //       <Provider store={store}>
    //         <ConfigProvider>
    //           <Router>
    //             <Suspense>
    //               <AdminDashboard />
    //             </Suspense>
    //           </Router>
    //         </ConfigProvider>
    //       </Provider>
    //     </I18nextProvider>
    //   )
    //   .toJSON()
    // expect(tree).toMatchSnapshot()
  })
})
