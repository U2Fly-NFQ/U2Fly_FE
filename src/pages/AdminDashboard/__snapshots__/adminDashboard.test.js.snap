// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Flight Chart renders correctly 1`] = `null`;
